### ISSUE TITLE
YAML meta data ignored

### ISSUE THREAD

**Issue by inventionate** [davidar/pandiff#10]
Thank you for your great package. Unfortunately there seems to be a problem with YAML blocks for meta information. I defined my title there, but it is ignored when creating the diff file (warning pushed and default empty title inserted) . I can't pass additional arguments either. Can this please be fixed? It is very unfavorable, if the title is not considered.

**Comment by davidar** [comment]
Are you expecting it to display a diff of the metadata blocks, or to just retain the metadata of the most recent document?

**Comment by inventionate** [comment]
Thank you very much for the response. My main concern is retain the metadata, although there are certainly cases where a comparison of the metadata is useful, if that's possible.

**Comment by marvinjuette** [comment]
I totally agree. I want to use this tool inside my ci pipeline to build both the final document and the diff view. For that, I would need to keep the YAML data of the new.md. But since the markdown seems to be converted to HTML during the diff highlight calculation the yaml data is lost, because the html has no concept for that kind of yaml support.

Therefore, I see two possible solutions:
1. don't convert the files to html anymore (bigger refactoring of the current application)
2. add a new argument with that the user can control if he wants to display metadata changes, or whether to use the old or new ones for the resulting diff document.

@davidar What of these solutions do you prefer? I like the second better. I probably have some spare time on the next weekend where I could implement the second option ^^
